client/src/components/Header/Header.jsx

```diff
@@ -1,21 +1,24 @@
 import React, { useContext } from 'react';
 import {
-  GameTitle, CategoryTree, CategoryWrapper, GameTitleWrapper,
+  GameTitle, CategoryTree, Wrapper, Category,
 } from './styles';
 import GamesContext from '../../context';
 
 const Header = () => {
   const { gameTitle, gameGenre } = useContext(GamesContext);
   return (
-    <>
+    <Wrapper>
       <CategoryTree>
```

Comment:
Really clean, solid use and code layout here with displaying the game hierarchy in separate lines, easy to read and looks nice